<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraCalibrationWidget</class>
 <widget class="QWidget" name="CameraCalibrationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>676</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CameraCalibrationWidget</string>
  </property>
  <widget class="QPushButton" name="pushButtonCalibration">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Calibration</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxDevice">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonChangeDevice">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Initialisation</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera Name</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxChessWidth">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="value">
    <number>7</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxChessHeigth">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="value">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelWidth">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHeigth">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Heigth</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelChessPattern">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Chess Pattern</string>
   </property>
  </widget>
  <widget class="QLabel" name="videoFrame">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Video Vorschau</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>3</number>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <item row="0" column="0">
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item row="2" column="2">
    <property name="text">
     <string>33</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="videoFrameUndistorted">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Video Vorschau
Entzerrt</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLoadCalib">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Calibration</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDeviceNo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>DeviceNo.</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxBilderNo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelBilderNo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Bilder</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
